eea/facetednavigation/upgrades/configure.zcml

```diff
@@ -5,6 +5,22 @@
   i18n_domain="eea">
 
   <genericsetup:upgradeSteps
+    source="7.5"
+    destination="8.4"
+    profile="eea.facetednavigation:default">
+
+     <genericsetup:upgradeDepends
```

Comment:
This will re-import all eea.facetednavigation profile.
```
 <genericsetup:upgradeDepends
       title="Import all steps from eea.facetednavigation default profile"
       import_profile="eea.facetednavigation:default"
       />
```
I don't think this is needed.